dokan_np/dokannp.c

```diff
@@ -562,8 +561,23 @@ DWORD APIENTRY NPEnumResource(__in HANDLE Enum, __in LPDWORD Count,
     return WN_NO_MORE_ENTRIES;
   }
 
+  DWORD processId = GetCurrentProcessId();//��ǰ���id  
```

Comment:
What does that comment mean?

Reply:
Hi @Rondom ，its a comment in Chinese,I remove it.please ignore it.